client/icons.go

```diff
@@ -26,17 +26,17 @@ import (
 	"regexp"
 )
 
-// Icon represents the icon of an installed Package
+// Icon represents the icon of an installed Snap
```

Comment:
These should probably be lowercased as well. We're talking more about snap-the-concept than actual types here and below. Sentence should also end on a full stop.